test/controllers/comment_controller_test.exs

```diff
@@ -64,6 +64,28 @@ defmodule CodeCorps.CommentControllerTest do
     test "does not create resource and renders 403 when not authorized", %{conn: conn} do
       assert conn |> request_create(@valid_attrs) |> json_response(403)
     end
+
+    @tag :authenticated
+    test "tracks comment created", %{conn: conn, current_user: current_user} do
```

Comment:
Fine by me. I was following the example set by the user controller tests [here](https://github.com/code-corps/code-corps-api/blob/b1906b2586da423089042c5b3b0139de2fcb94e7/test/controllers/user_controller_test.exs#L81) and [here](https://github.com/code-corps/code-corps-api/blob/b1906b2586da423089042c5b3b0139de2fcb94e7/test/controllers/user_controller_test.exs#L127). Should I consolidate those tests as well?